packages/gatsby-image/src/index.js

```diff
@@ -414,6 +414,11 @@ class Image extends React.Component {
       itemProp,
     }
 
+    if (this.props.onClick)
+      throw new Error(
+        `'onClick' event handlers are not allowed for accessibility reasons.`
+      )
+
```

Comment:
Code style: Please wrap sub-statements in a block if they do not stay on the same line. Prettier won't auto-fix this for you (although we should add a linting rule for this, if one isn't already in place).

Doing it without a block can be confusing in the JS world and in particular to cases where a newline is forbidden (`return` arg, `throw`, `yield`).

Reply:
Understood. I've gone ahead and wrapped it in a block. Is it alright to `throw` without `return`, or would it be best to change that too?

Reply:
A `throw` is effectively a hard `return` so that's not necessary :)